github/pulls.go

```diff
@@ -251,12 +251,16 @@ type PullRequestMergeResult struct {
 
 // PullRequestOptions lets you define how a pull request will be merged.
 type PullRequestOptions struct {
-	Squash bool
+	CommitTitle string
+	MergeMethod string
 }
 
 type pullRequestMergeRequest struct {
 	CommitMessage *string `json:"commit_message"`
-	Squash        *bool   `json:"squash,omitempty"`
+	CommitTitle   *string `json:"commit_title,omitempty"`
+
+	// The merge method to use. Possible values include: "merge", "squash", and "rebase" with the default being merge.
```

Comment:
While this is good, I think it would be much better to move this to the exported struct above (`PullRequestOptions`) so that the auto-generated `godoc`s will read better.